src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -90,6 +93,7 @@
             </div>
 
             <div v-if="selectedTab === modelOutputTabs.Comparison" id="comparison-container" class="col-md-12">
+                <loading-tab :loading="loading.comparison"/>
```

Comment:
So this won't get shown atm as we fetch the comparison data separately and this is done when calibration has completed. I think no problem in including this here though, we could set state to `loading` when we start fetching calibration data? Though perhaps at a later point, not sure if they are able to get to the comparison page before the data will have finished fetching.